### ISSUE TITLE
Messages in the log, after removing the plugin and devices

### ISSUE THREAD

**Issue by fparrav** [NorthernMan54/homebridge-mcuiot#35]
Currently migrate my esp8266 to [HAA](https://github.com/RavenSystem/esp-homekit-devices) for native support (thanks for the work on homebridge-mcuiot, it was one of my first approaches to homekit / homebridge), remove devices by validating ID in EVE app, and then remove homebridge-mcuiot from config .json, but these messages still appear in the log, even with the plugin uninstalled.

```
[5/3/2020, 3:05:50 PM] [mcuiot] Reading MCUIOT: NODE-80FD3D
[5/3/2020, 3:06:50 PM] [mcuiot] Reading MCUIOT: NODE-80F55D
[5/3/2020, 3:06:50 PM] [mcuiot] Reading MCUIOT: NODE-80FD3D
[5/3/2020, 3:07:50 PM] [mcuiot] Reading MCUIOT: NODE-80F55D
[5/3/2020, 3:07:50 PM] [mcuiot] Reading MCUIOT: NODE-80FD3D
[5/3/2020, 3:08:50 PM] [mcuiot] Reading MCUIOT: NODE-80F55D
[5/3/2020, 3:08:50 PM] [mcuiot] Reading MCUIOT: NODE-80FD3D
[5/3/2020, 3:09:50 PM] [mcuiot] Reading MCUIOT: NODE-80F55D
[5/3/2020, 3:09:50 PM] [mcuiot] Reading MCUIOT: NODE-80FD3D
[5/3/2020, 3:10:50 PM] [mcuiot] Reading MCUIOT: NODE-80F55D
[5/3/2020, 3:10:50 PM] [mcuiot] Reading MCUIOT: NODE-80FD3D
```

**Comment by NorthernMan54** [comment]
You need to remove the plugin code itself, and it will stop

It has a auto config feature if homebridge finds the code base.

**Comment by fparrav** [comment]
> You need to remove the plugin code itself, and it will stop


What does this mean ?  it is not in the `config.json` or in the path of `npm -g root`

**Comment by NorthernMan54** [comment]
There are multiple possible locations depending on the NPM command used to install and the account used.  I really can't say where it is in your system

Google shows this

https://stackoverflow.com/questions/5926672/where-does-npm-install-packages

**Comment by fparrav** [comment]
I searched the whole system for a reference to "Homebridge-mcuiot" and can't find it, I only have a single node version installed as shown in the screenshot and I only have the plugins installed in this directory ... it is strange.


<img width="1585" alt="node packages" src="https://user-images.githubusercontent.com/9319430/85097368-4e62e880-b1c5-11ea-8059-fa089e0dcceb.png">

**Comment by NorthernMan54** [comment]
Strange, did you also look in /usr/local?

**Comment by fparrav** [comment]
yes... there is no other folder node_modules throughout the system
